# EventSiteData.msg

geometry_msgs/Point position
int64 num_streets